src/standalone/Bucket/Bucket.jl

```diff
@@ -143,13 +150,13 @@ end
         date_ref::Union{DateTime, DateTimeNoLeap},
         t_start,
         Space::ClimaCore.Spaces.AbstractSpace;
-        infile_path = Bucket.cesm2_albedo_dataset_path(),
+        get_infile_path = Bucket.cesm2_albedo_dataset_path,
         varname = "sw_alb"
     ) where {FT}
 
 Constructor for the BulkAlbedoTemporal struct.
 The `varname` must correspond to the name of the variable in the NetCDF
-file specified by `infile_path`.
+file specified by `get_infile_path`.
```

Comment:
any more clarification needed that this is now a function that returns the path, and downloads the data?

Reply:
I updated the docstring, hopefully it's more clear now